config/settings/common.py

```diff
@@ -375,3 +375,4 @@
     )
 
 OSMCHA_URL = env('OSMCHA_URL', default='https://osmcha.org')
+OSM_API_USER_AGENT = {"User-Agent": "OSMCha osmcha-django v4"}
```

Comment:
IMO `"v4"` would ideally get read from `__version__.py` or similar somewhere in the repository, so we don't forget to update this string when v5 is released. But I grepped and it looks like the current version isn't actually stored anywhere in the codebase (just in the git tags) so I'm fine with hard-coding for now. I'll try to make a mental note that this is something that needs updating next time there's a major version bump. 🙂